### PR TITLE
Start page to match prototype v7

### PR DESCRIPTION
**Pull request by PhilipGlover**
- match prototype v7
- pass a IDV Implementation date to Start page
- other text changes since v5
- the date config `IDV_IMPLEMENTATION_DATE=20250901` - I've assumed that dates in config should be `YYYYMMDD`, hence the need to manipulate the date in the code here

config:
- https://github.com/companieshouse/docker-chs-development/pull/1123
- https://github.com/companieshouse/ecs-service-configs-dev/pull/391

IDVA3-1051

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/routers/utils/index.ts [hunk 1]
```diff
@@ -24,3 +24,12 @@ export function formatDateBorn (dateOfBirth: any, lang: string): string {
         return "Invalid date";
     }
 }
+
+export function internationaliseDate (date: string, lang: string): string {
+    try {
+        return Intl.DateTimeFormat(lang === "en" ? "en-GB" : lang, { dateStyle: "long" }).format(new Date(date));
+    } catch (error) {
+        console.error("Error internationalising date:", error);
```
Comment (hepsimo) [inline src/routers/utils/index.ts:32]: Best use `logger.error` instead

**Review by hepsimo — APPROVED** [review]
Changes look good. One console.log needs replacing

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/routers/utils/index.ts (hunk 1 above)
Comment (PhilipGlover) [reply src/routers/utils/index.ts]: doh, copy paste mistake from function above.  Ta

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**